### DISCUSSION TITLE
latest nightly build 1.2.4 multiple monitors not switching

### DISCUSSION THREAD

**Discussion by flashgordonAR** [rustdesk/rustdesk#6553] [Q&A] [answered]
Hi, having installed latest nightly build 1.2.4 in both Windows and Android phone, when trying to switch main and secondary monitors of windows device, switch does not happen.  Sometimes, it flickers and flashes between main and secondary monitors in the Android screen.  In stable 1.2.3 version, there is no issue.

**Comment by rustdesk** [comment, marked as answer]
Thanks, confirmed, bug of android 1.2.4, @fufesou will fix it.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/6554 please try out tomorrow's nightly build.

**Reply by flashgordonAR** [reply]
Hi there, latest build still not working. 

Main monitor OK but second monitor sometimes shows for almost 1 sec and returns to main monitor.

Regards,
Gabriel

**Comment by aannaannaa** [comment]
Not sure if same root issue, but similar outcome for me on ubuntu 22.04 today. Rustdesk installed unremarkably, but when launched all I saw was a bar at top with rustbelt written in it. Clicking it didn't open any window, right click yield quit or new window, which didn't do anything exept add additional option if clicked it again (to 'open' a windown that didn't open).

Resolution:  i opened it using sudo and got the normal looking window. duh, i guess.

:)anna

**Comment by flashgordonAR** [comment]
Just to comment that from another Windows device, to connect to the Windows device, I can switch screens, view them in separate windows, view both monitors at the same time.  Windows to Windows.

**Comment by fufesou** [comment]
https://github.com/rustdesk/rustdesk/pull/6575

> Sometimes, it flickers and flashes between main and secondary monitors in the Android screen.

The cause is also found and fixed in https://github.com/rustdesk/rustdesk/pull/6575

**Reply by flashgordonAR** [reply]
Hi, with today's build, the one recently released, the issue is solved.  Thanks!
